lib/hs300child.ts

```diff
@@ -93,7 +97,8 @@ export default class HS300child extends device {
   }
   async getRelayState() {
     const r = await this.getSysInfo();
-    return r.relay_state;
+
+    return r.children.find(ch => ch.id === this.child.id).state;
```

Comment:
Note that I am using an EP40 and not a HS300, however they seem to work the same for all other functions, however `relay_state` comes back as undefined instead of a proper value.